package.json

```diff
@@ -13,27 +13,19 @@
     "url": "https://github.com/plotly/react-plotly.js-editor/issues"
   },
   "scripts": {
-    "make:combined-translation-keys":
```

Comment:
Does your prettifier act on json as well?

Reply:
yes, it shouldn't? It was from the start..

Reply:
Hmmm I haven't configured prettier to handle JSON in this lib. Not sure which rule sets it would be applying. I was imagining only targeting stuff in `src`. It wouldn't be a bad thing to prettify the `json` but we don't have CI tests to test this so our styling could end up fighting without an arbiter.